spec/features/items/index_spec.rb

```diff
@@ -57,5 +76,59 @@
         expect(page).to have_css("img[src*='#{@dog_bone.image}']")
       end
     end
+
+    it 'has statistics when there are enough unique item_orders' do
+      merchant = create(:merchant, :with_items, item_count: 12)
+      create(:item_order, item: merchant.items[2], quantity: 6)
+      create(:item_order, item: merchant.items[1], quantity: 8)
+      create(:item_order, item: merchant.items[3], quantity: 4)
+      create(:item_order, item: merchant.items[0], quantity: 10)
+      create(:item_order, item: merchant.items[4], quantity: 3)
+
+      create(:item_order, item: merchant.items[5], quantity: 2)
+      create(:item_order, item: merchant.items[6], quantity: 1)
+      create(:item_order, item: merchant.items[7], quantity: 1)
+      create(:item_order, item: merchant.items[8], quantity: 1)
+      create(:item_order, item: merchant.items[9], quantity: 1)
```

Comment:
To my point above, it looks like all of these item_orders use distinct items. What happens if an item has multiple item_orders?

Reply:
I do like this setup approach you are using.